driver/driver.go

```diff
@@ -203,7 +226,13 @@ func (d *Driver) Stop() {
 	d.srv.Stop()
 	d.log.Info("Waiting for JWT refresher to finish")
 	d.quit <- true
+	d.quit <- true
```

Comment:
Sending a quit signal once should be enough right?

Reply:
Well there is two go routines that receive this signal
So hence I send it twice

Reply:
You can just close the channel
Reading from a closed channel will return the default value of the defined channel type

https://play.golang.org/p/qqHGDC1FP0p